pim/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "PIM",
```

Comment:
```suggestion
    "name": "Product Informations Management",
```

Reply:
Yes, with no "s" to "Information" I imagine ;)